translations/es.yaml

```diff
@@ -287,8 +287,9 @@ general:
   timedReleaseClearStartDate: "Quitar Fecha de Inicio de Lanzamiento "
   timedReleaseNoSchedule: "Disponible inmediatamente cuando publicado "
   timedReleaseOnlyEndDate: "Disponible antes de {endDate}"
-  timedReleaseOnlyStartDate: " Disponible después de {startDate}"
-  timedReleaseStartAndEndDate: " Disponible a partir de {startDate} hasta {endDate}"
+  timedReleaseOnlyStartDate: "Disponible después de {startDate}"
```

Comment:
Extra space at the start? Doesn't seem to be in the other translations.

Reply:
yes, that's why i removed the leading whitespace while at it. could have been a separate commit, i know.

Reply:
Oh, I totally read this commit wrong. Wow. Rookie move Johnson! I thought you were adding the whitespace!